package/PKGBUILD

```diff
@@ -0,0 +1,34 @@
+# Author: williamjcm <w.jcm59@gmail.com>
+# Contributor: mosra <mosra@centrum.cz>
+# Based on the packages/archlinux/PKGBUILD file by mosra <mosra@centrum.cz> and the MSYS2/MinGW PKGBUILD templates
+
+_realname=corrade
+pkgbase=mingw-w64-${_realname}-git
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}-git"
+pkgver=dev
+pkgrel=1
+pkgdesc="C++11/C++14 multiplatform utility library"
+arch=('any')
```

Comment:
Interesting, for Arch this would mean the package has no native binaries and so can run on both 32bit and 64bit. But looking at the other MSYS packages, all of them seem to use `any` as well. That means the MSYS package contains both a 32bit and a 64bit build?

Reply:
No.

My guess is that it allows both the i686 and x86_64 packages to be installed on both Win32 and Win64.

Reply:
Okay. Let's keep this consistent with the remaining MSYS packages, then.